eng/Directory.Build.Data.targets

```diff
@@ -116,5 +121,12 @@
            Text="When UseProjectReferenceToAzureClients=true all Azure.* references should be Project References, but the following are not [@(ShouldBeProjectReference)]" />
   </Target>
 
+   <Target Name="RunApiCompat" AfterTargets="CoreBuild" Condition="'$(RunApiCompat)' == 'true'">
+    <MSBuild
+      Projects="$(MSBuildThisFileDirectory)/ApiCompat/ApiCompat.csproj"
```

Comment:
@pakrym should we call an explicit target here? Otherwise it will call the default "Build" target.

Reply:
The default target is not build. see https://github.com/Azure/azure-sdk-for-net/pull/9568/files#diff-4fac5206bbbb0d60c0e3704a313cf582R1